### PR TITLE
Adding invitees table

### PR DESCRIPTION
**Pull request by anishaas**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 source 'https://rubygems.org'
-ruby '2.2.1'
+ruby '2.2.3'
```
Comment (jah2488) [inline Gemfile]: Uhoh. I don't think we want to change the ruby version, unless everyone in @tiy-austin-ror/rustworks is okay with this.

File: app/models/event.rb [hunk 2]
```diff
@@ -1,4 +1,5 @@
 class Event < ActiveRecord::Base
   belongs_to :room
   belongs_to :user
+  has_
```
Comment (jah2488) [inline app/models/event.rb]: Ah Syntax Error.

File: app/models/invitee.rb [hunk 3]
```diff
@@ -0,0 +1,4 @@
+class Invitee < ActiveRecord::Base
```
Comment (jah2488) [inline app/models/invitee.rb:1]: Why `Invitee` and not `Invite` ?

**Comment by mmdotz** [comment]
:+1:

File: app/views/invitees/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,29 @@
+<p id="notice"><%= notice %></p>
```
Comment (jah2488) [inline app/views/invitees/index.html.erb:1]: We should remove this and keep all flash messages in the application.html.

File: app/views/invitees/show.html.erb [hunk 5]
```diff
@@ -0,0 +1,14 @@
+<p id="notice"><%= notice %></p>
```
Comment (jah2488) [inline app/views/invitees/show.html.erb]: Ditto from above.

File: app/views/invites/index.html.erb [hunk 6]
```diff
@@ -0,0 +1,29 @@
+<p id="notice"><%= notice %></p>
```
Comment (jah2488) [inline app/views/invites/index.html.erb]: Ditto

File: app/views/invites/show.html.erb [hunk 7]
```diff
@@ -0,0 +1,14 @@
+<p id="notice"><%= notice %></p>
```
Comment (jah2488) [inline app/views/invites/show.html.erb]: Ditto

File: config/routes.rb [hunk 8]
```diff
@@ -1,4 +1,6 @@
 Rails.application.routes.draw do
+  resources :invitees
+  resources :invites
```
Comment (jah2488) [inline config/routes.rb]: `Invites` **and** `Invitees`?

**Comment by jah2488** [comment]
Fix those small things and this should be good to go.

File: app/models/event.rb (hunk 2 above)
Comment (jah2488) [inline app/models/event.rb]: Still gotta get this syntax error.

**Comment by jah2488** [comment]
You missed two notices. Get rid of those and done :D

**Comment by jah2488** [comment]
:+1:

**Pull request merged**